.studiorc

```diff
@@ -0,0 +1,288 @@
+}
+
+function start-sessionsrv {
+  hab svc start core/builder-sessionsrv --bind router:builder-router.default --bind datastore:builder-datastore.default
+}
+
+function start-worker {
+  hab svc start core/builder-worker --bind jobsrv:builder-jobsrv.default --bind depot:builder-api.default
+}
+
+function stop-builder {
+  stop-admin
+  stop-admin-proxy
+  stop-api
+  stop-api-proxy
+  stop-datastore
+  stop-jobsrv
+  stop-originsrv
+  stop-router
+  stop-scheduler
+  stop-sessionsrv
+  stop-worker
+}
+
+function stop-admin {
+  hab svc start core/builder-admin
+}
+
+function stop-admin-proxy {
+  hab svc start core/builder-admin-proxy
+}
+
+function stop-api {
+  hab svc start core/builder-api
+}
+
+function stop-api-proxy {
+  hab svc start core/builder-api-proxy
+}
+
+function stop-datastore {
+  hab svc start core/builder-datastore
+}
+
+function stop-jobsrv {
+  hab svc start core/builder-jobsrv
+}
+
+function stop-originsrv {
+  hab svc start core/builder-originsrv
+}
+
+function stop-router {
+  hab svc start core/builder-router
+}
+
+function stop-scheduler {
+  hab svc start core/builder-scheduler
+}
+
+function stop-sessionsrv {
+  hab svc start core/builder-sessionsrv
+}
+
+function stop-worker {
+  hab svc start core/builder-worker
+}
+
+function dev_docs {
+  cat <<DOCS
+Welcome to the development environment for Habitat & Builder.
+The following commands are available:
+
+1. Building Habitat
+  * build-habitat - build all Habitat components
+  * build-hab - build the Habitat CLI
+  * build-sup - build the Habitat Supervisor
+  * build-launcher - build the Habitat Launcher
+2. Building Habitat Web
+  * TED TODO: ELLIOT WILL WRITE THIS
+3. Building Builder
+  * build-builder - build all Builder components
+  * build-admin - build the Admin Gateway
+  * build-admin-proxy - build the Admin Gateway Proxy
+  * build-api - build the API Gateway
+  * build-api-proxy - build the API Gateway Proxy
+  * build-datastore - build Builder's Datastore
+  * build-jobsrv - build the JobSrv
+  * build-originsrv - build the OriginSrv
+  * build-router - build the RouteSrv
+  * build-scheduler - build the Scheduler
+  * build-sessionsrv - build the SessionSrv
+  * build-worker - build the Worker
+4. Running Builder
+  * start-builder - run all Builder services
+  * stop-builder - stop all Builder services
+  * start-{svc} - start a specfic Builder service. See the
+    above list of build targets for available services
+  * stop-{svc} - stop a specific Builder service
+DOCS
+}
+
+install-packages
+
+HAB_DEPOT_CHANNEL=unstable
+APP_HOSTNAME=localhost:3000
+GITHUB_API_URL=https://api.github.com
+GITHUB_WEB_URL=https://github.com
+GITHUB_CLIENT_ID=0c2f738a7d0bd300de10
+GITHUB_CLIENT_SECRET=438223113eeb6e7edf2d2f91a232b72de72b9bdf
+GITHUB_ADMIN_TEAM=1995301
+GITHUB_WORKER_TEAM=1995301
+SSL_CERT_FILE=$(hab pkg path core/cacerts)/ssl/cert.pem
```

Comment:
Cannot wait to purge this from the Earth once we have the next `core/openssl` rebuilt.